assets/hmr.js

```diff
@@ -23,6 +24,10 @@ class HotModuleState {
     this.disposeCallbacks.push(callback);
   }
   accept(callback = true) {
+    if (!this.isAccepted) {
+      socket.send(JSON.stringify({id: this.id, type: 'hotAccept'}));
+      this.isAccepted = true;
+    }
     if (this.isLocked) {
```

Comment:
I think we can get rid of isLocked, it was put in place to make sure that only accept functions in the first module were registered, and isAccepted seems to be doing that.

What this loses: The ability to call multiple `accept` calls in the file. This is less important now, but should be treated as a regression since we're planning to add dependency-accepting calls soon (see the discussion in the esm-hmr repo)

Reply:
I'm planning on restoring this functionality, when we rebuild a file due to `?mtime=` I'd like to put `hmrEnabled` to `false` since theoretically an accept could get removed